src/workloads/query/CollScanPredicateSelectivityLarge.yml

```diff
@@ -0,0 +1,26 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests the performance of conjunctive collection scan queries where the order of
+  predicates matters due to selectivity of the predicates.
+
+LoadConfig:
+  Path: ../../phases/query/CollScanPredicateSelectivity.yml
+  Parameters:
+    Database: CollScanPredicateSelectivityLarge
```

Comment:
Do you really care about the database names in the different workload versions? (Or is this only to speed up you own local testing)

Reply:
Thats seems reasonable.